programmers/학교습.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Vujahday+Script&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="학교습.css" />
+    <script src="https://kit.fontawesome.com/4865ddbb32.js" crossorigin="anonymous"></script>
+    <title>한결오빠가 준 연습과제</title>
+  </head>
+  <body>
+<container class = "con">
```

Comment:
```suggestion
<container class = "container">
```
con 이 container 를 의미하는 클래스명인 것 같은데 가독성을 위해서 명확히 명시해주세요